src/server/worker/server/api_server.go

```diff
@@ -15,6 +15,7 @@ import (
 type WorkerInterface interface {
```

Comment:
Where is this used?  I think we should be moving away from defining veneer interfaces to avoid using the gRPC interfaces.  I would rather see us write utility functions (maybe generic in `grpcutil`), to make dealing with the gRPC interfaces less annoying.

Reply:
This is more like a driver interface for the implementation of the worker gRPC server, although the current implementation of it is not called a driver. This extra level of indirection may not be worth it.